pavelib/quality.py

```diff
@@ -784,22 +764,33 @@ def _lint_output(linter, count, violations_list, is_html=False, limit=0):
     if diff_threshold > -1:
         percentage_string = u'--fail-under={0}'.format(diff_threshold)
 
+    pylint_files = get_violations_reports("pylint")
+    pylint_reports = u' '.join(pylint_files)
+    if not run_diff_quality(
+        violations_type="pylint",
+        reports=pylint_reports,
+        percentage_string=percentage_string,
+        branch_string=compare_branch_string,
+        dquality_dir=dquality_dir
+    ):
+        diff_quality_pass = False
+        failure_reasons.append('Pylint violations in the code that was added or changed.')
```

Comment:
nit: i found this wording a little odd. What do you think about something like: 'Pylint violation(s) were added or changed in this commit'? Same for eslint